src/generators/java/renderers/EnumRenderer.ts

```diff
@@ -49,6 +52,9 @@ ${this.indent(this.renderBlock(content, 2))}
     if (typeof value === 'string') {
       return `"${value}"`;
     }
+    if (typeof value === 'object') {
+      return `"${JSON.stringify(value).replace(/"/g, '\\"')}"`;
```

Comment:
Do we need two `\\` as I guess, one is for `"` char and the second is for what?

Reply:
`JSON.stringify` returns `{"test": "test"}` but we wrap it in `"` meaning they have to be escaped 🙂

Reply:
ahh... the second is for the first `\` 👌🏼